careers/faq.md

```diff
@@ -5,24 +5,40 @@ sidebar_label: FAQ
+
+#### Should I pay for Co-op or just go for internships?
+
+Do both! Co-op is an excellent way to land your first Co-op/internship if you are jsut entering the world of tech. This is because there are jobs exclusive to students; therefore, those job postigs have smaller pools of competition and the expectations of the job are more junior in nature. You will also have a dedicated team of coordinators working in partnership with you to find and secure these opportunities where employers are provided with subsidies to hire students (i.e. they get a tax credit when hiring Co-op students). If you complete all 3 required Co-op work terms, you will will graduate with at least 12 months of full-time, relevant work experience giving you an edge in the job market post-graduation. If at any point in the program you decide to withdraw from Co-op, remember that you can no longer gain access to positions eligible for registered Co-op students, so keep this in mind when reviewing other opportunities outside of Co-op.
+
+#### Can I apply for internships not on mySuccess while in Co-op?
+
+Yes! If you apply for jobs outside of Co-op provided job postings, you are conducting a Guided Job Search. When conducting a Guided Job Search, you may need to reach out to coop@uwindsor.ca before applying. Please see each situation below:
+
+Within Windsor/Essex County/Chatham-Kent (In-person role): Reach out to coop@uwindsor.ca before applying
+Outside Windsor/Essex County/Chatham-Kent (In-person role): Apply on your own
+Online (Remote role): Apply on your own
+
+Please remember though, if you are enrolled in Co-op, you only apply to jobs during your assigned Job Competition period. The reason being is that there are many administrative tasks that go on behind the scenes (checking work permits, confirming Co-op registration, etc.) to ensure students can enter placements legally and smoothly. Not doing so can result in your Co-op placement being denied by the Co-op department.
+
+#### Do grades matter?
+
+For most Co-op applications you must submit your transcript. Smaller/local companies amy look at grades. However, for opportunities outside of Co-op's postings, you will not need to submit your transcripts for 99% of applications.
```

Comment:
I think mentioning something about that even when required to submit a transcript, most jobs dont care and just use it as a way to be certain that you are enrolled